UI_UX/Frontend/src/components/Address.js

```diff
@@ -1,3 +1,31 @@
 import React from 'react';
+import { Loading } from 'carbon-components-react';
+import Jazzicon from 'react-jazzicon';
 
-export const Address = () => <div />;
+export const Address = ({ loadingAccountInfo, address }) => {
+  let loadingIcon = undefined;
+  if (loadingAccountInfo)

```

Comment:
Go for a more functional approach instead of flag based conditionals. For instance, you can do something like:
```js
const loadingIcon = loadingAccountInfo ? loadingAccountInfo : <div><Loading... /> </div>
```

Reply:
Furthermore, it's not clear to me what the loading icon does with `loadingAccountInfo` or whether this should be passed as a parameter. What is `loadingAccountInfo` and why it's needed to be passed to the component?

Reply:
fixed